packages/shared/src/utils/constants/contract.ts

```diff
@@ -190,7 +198,7 @@ export const getContractCategoriesConfig = () =>
         'Ins & Outs': { name: localize('Ins & Outs'), categories: ['end', 'stay'] },
         'Look Backs': { name: localize('Look Backs'), categories: ['lb_high_low', 'lb_put', 'lb_call'] },
         Digits: { name: localize('Digits'), categories: ['match_diff', 'even_odd', 'over_under'] },
-        Vanillas: { name: localize('Vanillas'), categories: [VANILLALONG.CALL, VANILLALONG.PUT] },
+        Vanillas: { name: localize('Vanillas'), categories: [VANILLALONG.CALL, VANILLALONG.PUT, VANILLALONG.FX] },
```

Comment:
@akmal-deriv if vanilla_fx includes `VANILLALONGCALL` and `VANILLALONGPUT`, can we just use `vanillalongcall` and `vanillalongput` configs without adding additional for financial? The `proposal` request is going to be the same, right? and probably, there will be no separate trade type like 'Vanilla Financial' to switch to, as I understand? What I mean is Maybe we no longer need to add `vanilla_fx` to this config anymore? Financial or not only matters in terms of UI in some components, right? It probably just enough to have a function like `isVanillaFxContract` (which you already have) to check for a selected contract type vanillalongput or vanillalongcall and whether or not a selected asset is from forex. What do you think? The main concern here is that when a user has switched to Vanilla Financial in this case you have now, we are likely to face the same issues which we had on Vanilla Synthetics that lead us to separation of `call` and `put`

Reply:
There was no need to add it in the first place, I just don't like passing lots of variables and importing unnecessary stores across the components. Every day our codebase looks worse and worse and it feels like an endless loop by now, so I added proper solution as this is the right place for changing UI through the config. Anyways, if it somehow breaks something else I can implement another thing.